src/stores/session.ts

```diff
@@ -0,0 +1,35 @@
+import { defineStore } from 'pinia';
+
+export interface ISessionState {
+  theme: string;
+}
+
+/**
+ * Holds the name of the store.
+ */
+const storeName: string = 'session';
+
+export const useSessionStore = defineStore(storeName, {
+  state: (): ISessionState => ({
```

Comment:
Please don't use arrow functions for the `state` method, because it most likely needs to be extended in most cases.